### PR TITLE
Add the prefix to BSL config map so that object stores can use it whe…

### PR DESCRIPTION
**Pull request by betta1**
…n initializing

Signed-off-by: Antony Bett <antony.bett@gmail.com>

### CODE REVIEW

**Comment by betta1** [comment]
@nrb thanks, I updated the comment to include prefix.

**Comment by nrb** [comment]
@betta1 Looks good - how time sensitive is this? I'm of the opinion this is a low-impact change that we could get in for v1.1 release. We don't officially have a feature freeze, either.

@skriss thoughts?

**Comment by betta1** [comment]
Thanks @nrb, yes I agree it will be great if we can get this in for v1.1 release since our plugin requires the prefix during initialization.

**Comment by skriss** [comment]
You'll need to add `prefix` here as well: https://github.com/heptio/velero/blob/master/pkg/cloudprovider/config.go#L30

Let's make sure to manually test this, but I'm fine merging it before v1.1 given the low impact.

File: pkg/cloudprovider/config.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 func ValidateObjectStoreConfigKeys(config map[string]string, validKeys ...string) error {
 	// `bucket` is automatically added to all object store config by
```
Comment (skriss) [inline pkg/cloudprovider/config.go]: pls update this comment, and the godoc comment on this function, to reference "prefix" as well.

**Comment by betta1** [comment]
Thanks @skriss, I added the prefix to list of valid keys and updated the godoc as well.

**Comment by skriss** [comment]
thanks - looks good, but i'm going to do a quick test as well

**Pull request merged**